consensus/consortium/v2/consortium.go

```diff
@@ -208,18 +208,18 @@ func (c *Consortium) Author(header *types.Header) (common.Address, error) {
 }
 
 // VerifyBlobHeader verifies the header of a blob block
-func (c *Consortium) VerifyBlobHeader(block *types.Block, blobs *[]kzg4844.Blob, proofs *[]kzg4844.Proof) error {
+func (c *Consortium) VerifyBlobHeader(block *types.Block, blobs []kzg4844.Blob, proofs []kzg4844.Proof) error {
 	if err := c.verifyVersionHash(block); err != nil {
 		return err
 	}
 	if c.skipBlobCheck(block.Header()) {
 		return nil
 	}
-	if len(block.Header().BlobCommitments) != len(*blobs) || len(block.Header().BlobCommitments) != len(*proofs) {
-		return fmt.Errorf("mismatching number of blobs, commitments, and proofs. blobs: %d, commitments: %d, proofs: %d", len(*blobs), len(block.Header().BlobCommitments), len(*proofs))
+	if len(block.Header().BlobCommitments) != len(blobs) || len(block.Header().BlobCommitments) != len(proofs) {
```

Comment:
We should create a variable `header := block.Header()` because block.Header will create a new copied header every time it is called

Reply:
fixed